content/v1.11/getting-started/provider-aws-part-2.md

```diff
@@ -472,9 +473,8 @@ spec:
 ```
 
 {{< hint "tip" >}}
-An _XRD_ is a Kubernetes _custom resource definition_.  
```

Comment:
I removed this because I interpreted it the first 2-3 times as saying an XRD and a CRD are the same thing. I later realized it was saying an XRD is defined by a CRD. This is true, but I suspect a little mind-bending. I figured we could gloss over this here, especially given `kubectl explain` is the better/recommended way to explore a schema than `kubectl describe crd`.